pkg/kubelet/image_manager_test.go

```diff
@@ -216,7 +216,7 @@ func TestFreeSpaceImagesInUseContainersAreIgnored(t *testing.T) {
 	spaceFreed, err := manager.freeSpace(2048)
 	assert := assert.New(t)
 	require.NoError(t, err)
-	assert.Equal(1024, spaceFreed)
+	assert.EqualValues(1024, spaceFreed)
```

Comment:
This also feels like it's own PR.  Basically, th eideal mega-PR touches *just* godeps and contrib/mesos